modules/rules_go/0.41.0/presubmit.yml

```diff
@@ -0,0 +1,31 @@
+matrix:
+  platform:
+    - centos7
+    - debian10
+    - ubuntu2004
```

Comment:
Should we use newer distributions like debian11 and ubuntu 2204?

Also add macos_arm64 because it's becoming more popular. There may not be much value of testing 3 Linux AMD 64 distributions, if the compute cost is a concern.

Reply:
@meteorcloudy for when you are back: Should we revisit the default set of CI platforms? I don't want to make changes that can cause Bazel CI to be overloaded, especially not the macOS arm64 runners.